GANs/stylegan/pretrained_celebahq.py

```diff
@@ -34,8 +42,8 @@ def main():
     Gs.print_layers()
 
     image_num = 10
```

Comment:
I think image_num has to be the value of the argument '--num-imgs'. The others LGTM